interface/controller/app_controller.go

```diff
@@ -0,0 +1,5 @@
+package controller
+
+type AppController struct {
+	Pokemon interface{ PokemonController }
```

Comment:
Are you planning on adding more interfaces to the Pokemon attribute? If not you could just remove the `interface` wrapper

Reply:
Addressed in a more recent commit, thank you.